test/z_cli_daemon_test.go

```diff
@@ -38,7 +51,7 @@ func (suite *PouchDaemonSuite) TestDaemonCgroupParent(c *check.C) {
 	// Must kill it, as we may loose the pid in next call.
 	defer dcfg.KillDaemon()
 
-	cname := "TestDaemonCgroupParent"
+	cname := GenerateCnameWithRandomSuffix("TestDaemonCgroupParent")
```

Comment:
why we use random suffix here?

Reply:
@fuweid the logical is fine for existent codes, but I had always got error in CI testing as follows:

```shell
15:01:07 FAIL: /home/tmp/root/180816Aug131534399982/src/github.com/alibaba/pouch/test/z_cli_daemon_test.go:32: PouchDaemonSuite.TestDaemonCgroupParent
15:01:07 
15:01:07 /home/tmp/root/180816Aug131534399982/src/github.com/alibaba/pouch/test/z_cli_daemon_test.go:55:
15:01:07     c.Fatalf("run container failed, err:%v", result)
15:01:07 ... Error: run container failed, err:
15:01:07 Command:  /usr/local/bin/pouch --host unix:///tmp/test/pouch/pouchd.sock run -d --name TestDaemonCgroupParent registry.hub.docker.com/library/busybox:1.28 top
15:01:07 ExitCode: 1
15:01:07 Error:    exit status 1
15:01:07 Stdout:   
15:01:07 Stderr:   Error: failed to run container: {"message":"container name: TestDaemonCgroupParent: already existed"}
```

I gave  a double check in actual test host, the previous container  with name "TestDaemonCgroupParent" indeed exists under the /tmp/test/pouch/containers/. we have several solutions:

1. add clean up script in CI configuration before testing
2. give a random container name w/o considering test env
3. the clean up codes are added in 2018-07-02, and the pervious containers are created before 2018-07-02, and so we can log in test host then remove these containers manually

You don't need to change any code for  item 1 and 2, but the dirty test env need to be cleaned up before testing.  What do you think?